### ISSUE TITLE
Remediate Security Vulnerability: CVE-2024-0727

### ISSUE THREAD

**Issue by hungtran84** [nginxinc/docker-nginx#878]
Hi team,
We detected [CVE-2024-0727](https://nvd.nist.gov/vuln/detail/CVE-2024-0727) in the most recent nginx:alpine3.18-slim with Twistlock scan.

We would appreciate it if these could be patched or remedied, if at all feasible, for reasons of compliance. Please contact me if you require further information.

**Comment by oxpa** [comment]
Hello,

Thank you for the report.

openssl issues should be handled by alpine maintainers, I guess.
As soon as the fix is available images will be automatically updated by Docker.
Also, the image for mainline nginx will be updated to alpine 3.19 when docker-library/official-images#16622 is merged.

Let me know if this assessment doesn't match your expectations. Otherwise I will close the issue later.

**Comment by yosifkit** [comment]
`CVE-2024-0727` was fixed in openssl `3.1.4-r5` (in Alpine 3.18) which is the version installed in the images: https://git.alpinelinux.org/aports/commit/main/openssl?h=3.18-stable&id=d2813a25a473f4b262b07900785a3531c9437a06
```console
$ docker run -it --rm nginx:alpine3.18-slim sh
/ # apk list | grep openssl
libcrypto3-3.1.4-r5 x86_64 {openssl} (Apache-2.0) [installed]
libssl3-3.1.4-r5 x86_64 {openssl} (Apache-2.0) [installed]
```
